website/docs/docs/build/metrics-overview.md

```diff
@@ -13,7 +13,10 @@ The keys for metrics definitions are:
 | Component | Description | Type |
 | --------- | ----------- | ---- |
 | `name` | Provide the reference name for the metric. This name must be unique amongst all metrics.   | Required |
-| `type` | Define the type of metric, which can be a measure (`simple`) or ratio (`ratio`)).  | Optional |
+| `description` | Provide the description for your metric.   | Optional |
+| `configs` | Provide the specific configurations for your metric   | Optional |
```

Comment:
@dave-connors-3 I like this idea! Should we also move configs and label down below `type_params`?